heidegger_index/urls.py

```diff
@@ -2,20 +2,47 @@
 from django.views.generic import RedirectView
 from django.conf import settings
 
-from heidegger_index.views import LemmaDetailView, index_view, WorkDetailView, WorkDetailViewMD, LemmaDetailViewMD, URNRedirectView
+from heidegger_index.views import (
+    LemmaDetailView,
+    index_view,
+    WorkDetailView,
+    WorkDetailViewMD,
+    LemmaDetailViewMD,
+    URNRedirectView,
+)
 
 namespaced_patterns = (
     [
         path("", index_view, name="home"),
-        path("work/<slug>.md", WorkDetailViewMD.as_view(content_type='text/markdown'), name="work-md-export"),
-        path("lemma/<slug>.md", LemmaDetailViewMD.as_view(content_type='text/markdown'), name="lemma-md-export"),
-        path("work/<slug>", WorkDetailView.as_view(), name="work-detail"),
+        path(
+            "work/<slug:slug>/",
+            include(
+                [
+                    path("", WorkDetailView.as_view(), name="work-detail"),
+                    path(
+                        ".md",
```

Comment:
Werkt dit? Het lijkt of de url hiermee veranderd is in `work/<slug>/.md`

Verder overigens mooie suggestie om sub-URL's te definiëren.

Reply:
Ja, je hebt gelijk. Het was mij nog niet opgevallen omdat we die url eigenlijk alleen maar _reversed_ gebruiken. Ik zal het even corrigered